avatar.html

```diff
@@ -0,0 +1,113 @@
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+    />
+    <link rel="icon" href="./images/logo.png" type="image/icon type" />
+    <title>Mettl-UI | Docs</title>
+  </head>
+
+  <body>
+    <nav class="navbar navbar-horizontal ">
+      <div class="nav-brand">
+        <img src="./images/logo.png" />
+        <a class="header link" href="/">Mettl UI</a>
+      </div>
+      <ul class="list-non-bullet">
+        <li class="list-item-inline">
+          <a class="link" href="/">Home</a>
+        </li>
+        <li class="list-item-inline link-active">
+          <a class="link" href="/document.html">Docs</a>
+        </li>
+        <li class="list-item-inline">
+          <a class="link" href="/download.html">Download</a>
+        </li>
+      </ul>
+    </nav>
+    <div class=" flex">
+      <aside class="aside-nav">
+        <ul class="list-item-non-bullet border-right">
+          <li class="navbar-vertical-item">
+            <a class="aside-link aside-current-item" href="./avatar.html"
+              >Avatar</a
+            >
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./alert.html">Alert</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./badge.html">Badge</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./button.html">Button</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./card.html">Card</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./image.html">Images</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./input.html">Input</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./list.html">Lists</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./navigation.html">Navigation</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./modal.html">Modal</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./rating.html">Rating</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./utilities.html">Text utilities</a>
+          </li>
+          <li class="navbar-vertical-item">
+            <a class="aside-link" href="./toast.html">Toast</a>
+          </li>
+        </ul>
+      </aside>
+      <main class="main ">
+        <h1 class="heading-primary">Avatar</h1>
+
+        <p class="heading-primary fs1">
+          You can select avatars with image based on size:
+        </p>
+        <div class="main-content">
+          <div class="component-container  pd2">
```

Comment:
Hi Pooja,
Class names (fs1, pd2) can be changed to something more descriptive

Reply:
done